### ISSUE TITLE
Specifying multiple static dirs by using commas fails if a space is used after comma.

### ISSUE THREAD

**Issue by cramatt** [storybooks/storybook#2996]
## tldr;

Fails: 
```
build-storybook -s ./app, __mock_stores__, ./assets -c .storybook -o .out
```
Works: 
```
build-storybook -s ./app,__mock_stores__,./assets -c .storybook -o .out
```

Notice no spaces after commas in second example? If this is desired, I would suggest making it clear in docs. 

-----

> Originally I thought the issue was this: 

### Issue details

I'm attempting to run build-storybook in my project, using static files option, and get an error that my static files directory does not exist - funny thing is, it does exist, and this error happens after all the directory contents are already copied from dir -> built storybook dir. 

Here are my logs: 

```
mmiller$ yarn storybook-build
yarn run v1.3.2
$ build-storybook -s ./app, __mock_stores__, ./assets -c .storybook -o .out
info @storybook/react v3.3.13
info
info => Loading custom .babelrc
info => Loading custom addons config.
info => Loading custom webpack config (full-control mode).
info => Copying static files from: ./app
ERR! Error: no such directory to load static files:
error Command failed with exit code 255.
```

Here is a screenshot showing the static files dir, `app`, and the files copied over to the output dir, `.out`. After this step, the command fails. 

<img width="307" alt="screen shot 2018-02-15 at 6 20 21 pm" src="https://user-images.githubusercontent.com/22599518/36290938-11fae102-127d-11e8-8599-bdecaa36af49.png">
<img width="322" alt="screen shot 2018-02-15 at 6 20 04 pm" src="https://user-images.githubusercontent.com/22599518/36290939-1216e6f4-127d-11e8-9266-b97bbafb4cde.png">

### Steps to reproduce

- Create a storybook project
- Add some static folder with assets
- Try to run build command. 

If this is not a silly misconfiguration error on my part, then I suspect I'll need to put together a more detailed repo to demo this...

### Please specify which version of Storybook and optionally any affected addons that you're running

```
"@storybook/addon-console": "1.0.0",
"@storybook/addon-knobs": "3.3.13",
"@storybook/addon-options": "3.3.13",
"@storybook/react": "3.3.13",
```

Node v8.9.3
Npm 5.5.1

### Affected platforms 

Mac OS 

### Screenshots / Screencast / Code Snippets (Optional)

n/a

**Comment by cramatt** [comment]
Ok, I have more info on this. What's actually happening is the build command is trying to copy files from a directory called ` ` (space) which does not exist. You don't see this in the error message because:

```
ERR! Error: no such directory to load static files:  <---- there is a space character here which is easy to miss!!! 
```

I misread this as `./app` being the problem folder. 

So, I would suggest updating this log to have a period, so the log would have read for me: 

```
ERR! Error: no such directory to load static files:  .
```

But that is obviously still easy to miss.

The way I diagnosed this was by adding a log statement here: https://github.com/storybooks/storybook/blob/a78016c4c9ba9defdd4a5ef86cdf6bbc78e83425/lib/core/src/server/build-static.js#L61-L70

Which logs the following: 

```
info => Loading custom .babelrc
info => Loading custom addons config.
info => Loading custom webpack config (full-control mode).
[ './app', '' ]
```

Which doesn't seem right since `-s ./app, ./__mock_stores__, ./assets` works (or at least appears to) when running storybook server. 

The docs https://storybook.js.org/configurations/serving-static-files/ show that a comma can be used to specify multiple dirs, but they are not clear that a space CANNOT be used between each folder. 

If I modify the command to be `-s ./app,./__mock_stores__,./assets` the issues seems to be resolved.

```
info => Loading custom .babelrc
info => Loading custom addons config.
info => Loading custom webpack config (full-control mode).
[ './app', './__mock_stores__', './assets' ]
```

**Comment by Hypnosphi** [comment]
Are there examples of other CLI tools that accept secondary parameters as space-separated lists without quotes around it?

**Comment by igor-dv** [comment]
I think it should also work with multiple -s params: `-s ./app, -s ./__mock_stores__, -s ./assets`

**Comment by danielduan** [comment]
I think we can solve this by clarifying it to our documentation. Either comma separated with no spaces or multiple `-s`. Any of you want to open a ticket for it?

**Comment by danielduan** [comment]
Doc updated.
